sdv/multi_table/base.py

```diff
@@ -524,9 +528,18 @@ def sample(self, scale=1.0):
             total_columns += len(table.columns)
 
         table_columns = getattr(self, '_original_table_columns', {})
+        if not hasattr(self, '_original_metadata'):
+            self._original_metadata = self.metadata
```

Comment:
Instead of setting this attribute here, maybe we should just use `self.get_metadata`